.benchmark/main.cpp

```diff
@@ -65,25 +65,30 @@ void benchmark_lambda_call()
 
 int main()
 {
+    std::cerr << "This executable is not intended for end-users and should not be used. To verify jluna works, instead execute \"./JLUNA_TEST\"" << std::endl;
+    return 1;
+
     using namespace jluna;
     State::initialize();
 
+    auto test = Main["abc"];
```

Comment:
[]() you have my money

Reply:
- - [ ] > > ~~@Clemapfel ![IMG_20220303_071645466.jpg](https://user-images.githubusercontent.com/100173891/156670495-d25d6484-ee72-43fb-a460-c2e028d8072b.jpg)~~